src/main/scala/ru/org/codingteam/keter/ui/views/MenuView.scala

```diff
@@ -0,0 +1,41 @@
+package ru.org.codingteam.keter.ui.views
+
+import org.scalajs.dom.KeyboardEvent
+import ru.org.codingteam.keter.ui.IView
+import ru.org.codingteam.keter.ui.viewmodels.MenuViewModel
+import ru.org.codingteam.keter.util.Logging
+import ru.org.codingteam.rotjs.interface.{Display, ROT}
+import ru.org.codingteam.rotjs.wrappers._
+
+class MenuView(viewModel: MenuViewModel) extends IView with Logging {
+
+  private val keyMap = Map[Int, () => Unit](
+    ROT.VK_UP -> viewModel.up _,
+    ROT.VK_DOWN -> viewModel.down _,
+    ROT.VK_RETURN -> viewModel.execute _
+  )
```

Comment:
This mapping should be exposed somehow (i'd do this through an abstract method) to a base View class, which wil handle all the input processing and drop the result back into concrete implementation (through another abstract method, mayhaps? (yes, I fucking LOVE abstract methods (and default implementations of interface methods in J8)))